src/main/java/com/google/devtools/build/lib/rules/apple/ApplePlatform.java

```diff
@@ -53,7 +53,7 @@
   private static final ImmutableSet<String> TVOS_DEVICE_TARGET_CPUS =
       ImmutableSet.of("tvos_arm64");
   private static final ImmutableSet<String> MACOS_TARGET_CPUS =
-      ImmutableSet.of("darwin_x86_64");
+      ImmutableSet.of("darwin_x86_64", "darwin");
```

Comment:
can you add a note here on why `darwin` is present? Perhaps linking to the other PR? Just so it's easier to follow what happened when someone takes the cleanup task of this.

Reply:
Done